scripts/resetdeps.sh

```diff
@@ -9,3 +9,9 @@ git checkout node_modules
 node . i --ignore-scripts --no-audit --no-fund "$@"
 node . rebuild --ignore-scripts
 node . run dependencies --ignore-scripts
+# check for cmark-gfm bindings
+cmarkbinding=$(find `npm ls cmark-gfm --parseable \
```

Comment:
i think this might need to use `node .` like the rest of the commands to ensure it uses the local repo and not the globally installed version while in CI

Reply:
Oh nuts, I didn't catch that